templates/_helpers.yaml

```diff
@@ -276,3 +276,7 @@ Create the name of the cleanup service account to use
     {{ default "default" .Values.airflow.cleanup.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{- define "astro.registry_secret" -}}
+  {{- default (printf "%s-registry" .Release.Name) .Values.airflow.registry.secretName }}
+{{- end }}
```

Comment:
I added this change to the other PR with the removal of all the 0.31 features https://github.com/astronomer/airflow-chart/pull/427 so we can remove it here.

```suggestion
```